stdlib/cmake/modules/SwiftSource.cmake

```
@@ -457,6 +457,14 @@ function(_compile_swift_files
     list(APPEND swift_flags "-Xfrontend" "-sil-verify-all")
   endif()
 
+  if(SWIFT_SIL_VERIFY_ALL_MACOS_ONLY)
+    # Only add if we have a macOS build triple
+    if (STREQUAL "${SWIFTFILE_SDK}" "OSX" AND
+        STREQUAL "${SWIFTFILE_ARCHITECTURE}" "x86_64")
```

From the help it seems `STREQUAL` is an infix operator (https://cmake.org/cmake/help/latest/command/if.html#comparisons), so I'm afraid this may fail if we enable this in a preset

```suggestion
    if ("${SWIFTFILE_SDK}" STREQUAL "OSX" AND
          "${SWIFTFILE_ARCHITECTURE}" STREQUAL "x86_64")
```